backend/src/main/java/com/example/vehiclesservice/controller/GaragesController.java

```diff
@@ -0,0 +1,20 @@
+package com.example.vehiclesservice.controller;
+
+import com.example.vehiclesservice.api.Garage;
+import com.example.vehiclesservice.service.GarageService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/garage", produces = MediaType.APPLICATION_JSON_VALUE)
+@AllArgsConstructor
+public class GaragesController {
+
+    private final GarageService garageService;
+
+    @GetMapping("/getAll")
```

Comment:
You should use just /garages for getAll.
Not necessary to change, just to mention some convention :)
GET       /garages - get all garages
GET       /garages/{id} - get garage by id 
POST     /garages - add new garage
PUT       /garages/{id} - update garage
DELETE  /garages/{id} - delete garage ...